config/routes.rb

```diff
@@ -2,7 +2,8 @@
 
 Rails.application.routes.draw do
   mount Sidekiq::Web => '/sidekiq'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  mount ActionCable.server => "/cable"
```

Comment:
For my own knowlege, is "/cable" the open channel or 'endpoint address' that FE will establish a connection with?

Reply:
That's a solid question and I honestly don't fully know the answer. I followed a tutorial for the set-up and tried working my way through how we'll need to change the set-up for our unique needs. 

It's my understanding that we won't be creating a specific endpoint for the API with that route, but it's the the way that we can utilize the built-in ActionCable functionality to establish the open connection. On the FE, they'll hook into that route using a WebSocket specific protocol.